app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -60,8 +60,13 @@ def self.raw_create_cloud_network(ext_management_system, options)
   end
 
   def raw_delete_cloud_network
-    ext_management_system.with_provider_connection(connection_options(cloud_tenant)) do |service|
-      service.delete_network(ems_ref)
+    with_notification(:cloud_network_delete,
+                      :options => {
+                        :subject => self,
```

Comment:
When an object is passed as `:subject` like this, does it get translated nicely into a string for the notification message, or does it show up as an object reference?

Reply:
This is how the rest of these notifications are written, so I assume it's fine